qiskit/transpiler/passes/calibration/rx_builder.py

```diff
@@ -0,0 +1,116 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Add single-pulse RX calibrations that are bootstrapped from the SX calibration."""
+
+from typing import Union
+from functools import lru_cache
+import numpy as np
+
+from qiskit.circuit import Instruction
+from qiskit.pulse import Schedule, ScheduleBlock, builder, DriveChannel
+from qiskit.pulse.channels import Channel
+from qiskit.pulse.library.symbolic_pulses import Drag
+from qiskit.transpiler.passes.calibration.base_builder import CalibrationBuilder
+from qiskit.transpiler import Target
+from qiskit.circuit.library.standard_gates import RXGate
+from qiskit.exceptions import QiskitError
+
+
+class RXCalibrationBuilder(CalibrationBuilder):
+    """Add single-pulse RX calibrations that are bootstrapped from the SX calibration.
```

Comment:
It's worth having an example code, i.e. `.. code-block:: python`.

Reply:
I see. I added a quick demo of this pass.

``` python
backend = FakeBelemV2()
pm = PassManager(RXCalibrationBuilder(backend.target))
qc = QuantumCircuit(1)
angles = [0.1, 0.2, 0.3, 0.4]
for angle in angles:
     qc.rx(angle, 0)
```